content/browser/webui/brave_shared_resources_data_source.cc

```diff
@@ -162,9 +167,14 @@ BraveSharedResourcesDataSource::GetAccessControlAllowOriginForOrigin(
   // back.
   std::string allowed_origin_prefix = content::kChromeUIScheme;
   allowed_origin_prefix += "://";
+  std::string allowed_untrusted_origin_prefix =
+      content::kChromeUIUntrustedScheme;
+  allowed_untrusted_origin_prefix += "://";
   if (!base::StartsWith(origin, allowed_origin_prefix,
+                        base::CompareCase::SENSITIVE) &&
+      !base::StartsWith(origin, allowed_untrusted_origin_prefix,
                         base::CompareCase::SENSITIVE)) {
-    return "null";
+    return std::string();
```

Comment:
this is technically an invalid value for CORS according to the spec, but it should be fine security-wise.

Reply:
Empty string instructs chromium to omit the header. This is what the super class expects.

Reply:
👍